app/models/contract.rb

```diff
@@ -53,12 +53,16 @@ class Contract < ApplicationRecord
   # TODO: unit test this scope
   def self.provided_by(account)
     where.has do
-      plan_id.in(Plan.provided_by(account).select(:id))
+      # NOTE: `reorder(nil)` resets the order set in the Plan's default scope,
+      # because ORDER BY in subqueries fails in Oracle with 'ORA-00907: missing right parenthesis'.
+      plan_id.in(Plan.provided_by(account).select(:id).reorder(nil))
```

Comment:
Better remove the default scope than having to do this hack elsewhere. I think.

Reply:
But then, probably some other parts of the application rely on the default scope, and removing it may break them, and we need to fix it in all those places :sweat_smile: 

But we can check, maybe there are not so many...

Reply:
leave it for another PR maybe

Reply:
> But then, probably some other parts of the application rely on the default scope, and removing it may break them, and we need to fix it in all those places 😅

Yeah, if that's the case better do it in another PR.

Reply:
I'm doing it here: https://github.com/3scale/porta/pull/3784